appveyor.yml

```diff
@@ -183,13 +185,25 @@ for:
       - sh: unset LD_LIBRARY_PATH
       - sh: cp ../../setup/unix/UltraStar-Creator.desktop .
       - sh: cp ../../setup/unix/UltraStar-Creator.png .
+      - sh: sed -i "s/Name=UltraStar-Creator/Name=LINUX-UltraStar-Creator/g" UltraStar-Creator.desktop 
       - sh: ./linuxdeployqt*.AppImage UltraStar-Creator.desktop -bundle-non-qt-libs -appimage
 
     artifacts:
-      - path: bin/release/UltraStar-Creator-*.AppImage
-        name: UltraStar-Creator AppImage Package
+      - path: bin/release/LINUX-UltraStar-Creator-*.AppImage
+        name: LINUX-UltraStar-Creator-appimage
         
     on_failure:
       # Change false to true to block build and investigate build failures
       - sh: export APPVEYOR_SSH_BLOCK=true
       - sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e -
+
+deploy:
+  description: ''
+  provider: GitHub
+  draft: true
+  auth_token:
+    secure: ykwnM89TZTiQ/nmI5ijPtlekJ3BgzaT37M/hYeMNYnUvPO1U+JoYIWLXjHcZCSyL
```

Comment:
You'll have to change this to your own Github API token which you can generate [here](https://github.com/settings/tokens)
Make sure to [hash it through AppVeyor](https://ci.appveyor.com/tools/encrypt) before commiting it.